packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -270,7 +271,10 @@ export class Autosuggest {
   private text = i18n(() => this.host, translations);
 
   componentDidRender() {
-    this.setListboxContainerMaxBlockSize();
+    // This needs to be debounced to avoid a race condition between render() and componentDidRender() whereby
+    // the height of listitems might alternate between two values due to the (dis)appearing vertical scrollbar of the
+    // scrollable's scroll-container
+    debounce(() => this.setListboxContainerMaxBlockSize(), 150);
```

Comment:
```suggestion
  private hasCalculatedHeight = false;

  componentDidRender() {
    // This needs to be debounced to avoid a race condition between render() and componentDidRender() whereby
    // the height of listitems might alternate between two values due to the (dis)appearing vertical scrollbar of the
    // scrollable's scroll-container
    if (!this.hasCalculatedHeight) {
      setTimeout(() => {
        this.setListboxContainerMaxBlockSize();
        this.hasCalculatedHeight = true;
      }, 150);
    }
  }
```

Zo trapt stencil de functie maar 1 maal af na render